react-main/src/app/screens/Game/components/Moves/index.js

```diff
@@ -22,4 +24,11 @@ class Moves extends Component {
   }
 }
 
+Moves.propTypes = {
+  history: PropTypes.arrayOf(PropTypes.objectOf(PropTypes.arrayOf(PropTypes.string))).isRequired,
+  jumpTo: PropTypes.func.isRequired,
+  winner: PropTypes.string,
+  xIsNext: PropTypes.bool
```

Comment:
xIsNext should be required, since it is created by your app with false value, and never gets null value